ptx/sec_IBP.ptx

```diff
@@ -1255,7 +1256,7 @@
               $f = Formula("(ln(x+$b))^2")->reduce;
               Context("Fraction");
               Context()->variables->set(x=>{limits=>[-$b,-$b+4]});
-              $F = FormulaUpToConstant("(x+$b)*(ln(x+$b)^2-2*ln(x+$b))")->reduce;
+              $F = FormulaUpToConstant("2x+(x+$b)*ln(x+$b)^2-2*(x+$b)*ln(x+$b)")->reduce;
```

Comment:
```suggestion
              $F = FormulaUpToConstant("2x+(x+$b)*ln(x+$b)^2-2*(x+$b)*ln(x+$b)")->reduce;
              $F->{limits} = [-$b+1,-$b+5];
```

Reply:
This is independent from your correction, which is good. But I noticed that with the randomization allowing `$b` to be something like `-9`, and with the default test point domain being `[-2,2]`, that could lead to not test points producing output. And then you get the "not enough test points" error message.

Reply:
Does the solution look nicer if `2x` is replaced with `2(x+$b)`? Then it can even look more like it was, but have a `+2` inside the parentheses. 

No opinion on this, just observations.